tests/robot-tests/tests/general_public/methodologies_page.robot

```diff
@@ -1,44 +1,40 @@
 *** Settings ***
 Resource            ../libs/public-common.robot
-Force Tags          GeneralPublic    Local    Dev    Test    Preprod    Prod
+Force Tags          GeneralPublic    Prod
 
 Suite Setup         user opens the browser
 Suite Teardown      user closes the browser
 
 *** Test Cases ***
 Navigate to /methodology page
-    [Tags]    NotAgainstDev    NotAgainstPreProd    NotAgainstProd
+    [Tags]    Local    Prod
```

Comment:
The `Prod` tag here is redundant - so it can be removed on this test case and any other test case it appears on. 

As the `Prod` tag is in `Force Tags` it is applied to every single test case. So any tag included in `Force Tags` doesn't need to appear on any test case.